src/DivConformingFESpaces.jl

```diff
@@ -0,0 +1,77 @@
+function FESpaces.FESpace(model::DistributedDiscreteModel,
```

Comment:
This is not general enough (see below), but we can add it for the moment since we want to get things done.

E.g. one could build a FESpace with an instance of `GenericRefFE{RaviartThomas}` and you also want to compute the  globally consistent sign flip in this case.

Reply:
Yes, I was aware of this. I didnt do it before because I did not want to redo work. Now that we have agreed how to proceed, I will code it.

Reply:
Done